Source/WebKit/UIProcess/API/Cocoa/_WKInspectorExtension.h

```diff
@@ -75,6 +75,14 @@ WK_CLASS_AVAILABLE(macos(12.0))
  */
 - (void)evaluateScript:(NSString *)scriptSource inTabWithIdentifier:(NSString *)tabIdentifier completionHandler:(void(^)(NSError * _Nullable, id result))completionHandler;
 
+/**
+ * @abstract Navigates a tab created by this _WKInspectorExtension to a new URL.
+ * @param url The url to be loaded.
+ * @param tabIdentifier Identifier for the Web Inspector tab in which to navigate.
+ * @param completionHandler A block to invoke when the operation completes or fails.
+ */
+- (void)navigateToURL:(NSURL *)url inTabWithIdentifier:(NSString *)tabIdentifier completionHandler:(void(^)(NSError * _Nullable))completionHandler;
```

Comment:
I feel like `-navigateTabWithIdentifier:toURL:completionHandler:` would be clearer to read and would better match the naming/pattern of `-createTabWithName:tabIconURL:sourceURL:completionHandler:` (and `navigateExtensionTabWithIdentifier:toURL:completionHandler:`)

Reply:
I kept this as-is since it better matches the other methods with `inTabWithIdentifier:` being second.